internal/server/grpc/grpc.go

```diff
@@ -0,0 +1,97 @@
+package grpc
```

Comment:
Since this file implements `Server`, consider renaming it to `server.go` (so we have `/internal/server/grpc/server.go`).